money.js

```diff
@@ -0,0 +1,42 @@
+const PRECISION = 3
+const EXPONENT = (10 ** PRECISION)
+
+class Money {
+    constructor (amount, currencyCode) {
+       const decimalasStr = amount.toString().split('.')[1]
+       if (decimalasStr && decimalasStr.length> PRECISION) {
+           throw new Error('Maximum Money precision is' + PRECISION)
+       }
+       
+       this._amount = amount * EXPONENT
+       this.currencyCode = currencyCode
+      
+       plus (other) ;{
```

Comment:
What's up with these semicolons before the bracket? I don't believe that should work.